.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Publish
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        node-version: [lts/*]
+    
+    runs-on: ${{ matrix.os }}
+    
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install Linux Dependencies
+      if: runner.os == 'Linux'
+      run: sudo apt-get update && sudo apt-get install -y realpath
+
+    - name: Download Dependencies
+      run: ./build/download.sh
+
+    - name: Build
+      run: ./build/make.sh
+
+    - name: Publish Linux Package
```

Comment:
As this is written right now (and was before!), it will publish the Linux package even if the macOS build has failed.

We should consider splitting the publishing into another job that only runs if all previous jobs have succeeded.